Rakefile

```diff
@@ -1,5 +1,25 @@
+require 'json'
+require 'net/https'
 require 'rake/testtask'
 
+task :download do
+  LICENSES_URI = URI.parse('https://spdx.org/licenses/licenses.json')
+
+  http = Net::HTTP.new(LICENSES_URI.host, LICENSES_URI.port)
+  if LICENSES_URI.scheme == 'https'
+    http.use_ssl = true
+    http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Actually it doesn't work, I've tried it:

    OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed